### ISSUE TITLE
No such file or directory ~/faasm/results/runtime-bench-time.csv

### ISSUE THREAD

**Issue by bushidocodes** [lsds/Faasm#115]
When I run `inv bench-time`, I error out because this file/directory doesn't exist

```
inv bench-time
Traceback (most recent call last):
  File "/usr/local/bin/inv", line 8, in <module>
    sys.exit(program.run())
  File "/usr/local/lib/python3.6/dist-packages/invoke/program.py", line 384, in run
    self.execute()
  File "/usr/local/lib/python3.6/dist-packages/invoke/program.py", line 566, in execute
    executor.execute(*self.tasks)
  File "/usr/local/lib/python3.6/dist-packages/invoke/executor.py", line 129, in execute
    result = call.task(*args, **call.kwargs)
  File "/usr/local/lib/python3.6/dist-packages/invoke/tasks.py", line 127, in __call__
    result = self.body(*args, **kwargs)
  File "/usr/local/code/faasm/tasks/bench_time.py", line 22, in bench_time
    csv_out = open(OUTPUT_FILE, "w")
FileNotFoundError: [Errno 2] No such file or directory: '/home/sean/faasm/results/runtime-bench-time.csv'
```

I was able to get past this by running the following:

```
mkdir ~/faasm/results/
touch /home/sean/faasm/results/runtime-bench-time.csv
```

**Comment by Shillaker** [comment]
This should also be fixed in #117

**Comment by Shillaker** [comment]
Thanks so much for the detail on these @bushidocodes , as always keep 'em coming if you find any more issues!

Cheers
